.github/CODEOWNERS

```diff
@@ -33,7 +33,7 @@ go.sum @ajm188 @deepthi @harshit-gangal @mattlord @rohit-nayak-ps @systay @froui
 /go/test/endtoend/vtorc @deepthi @shlomi-noach @GuptaManan100 @timvaillancourt
 /go/tools/ @frouioui @systay
 /go/vt/dbconnpool @harshit-gangal @mattlord
-/go/vt/discovery @deepthi @frouioui
+/go/vt/discovery @deepthi @frouioui @timvaillancourt
```

Comment:
I wanted to point out that being a code owner is a responsibility. Meaning that you will be selected as a reviewer on PRs that modify this code and you are expected to review those PRs in a timely fashion as an owner of that code/area. I'm noting this as I have not seen you very active as a reviewer for the other areas in which you are already listed as a code owner. I would recommend not adding yourself to more areas until you can be more involved with reviews (it takes a lot of time and effort).